pkgs/by-name/fr/freediameter/package.nix

```
@@ -0,0 +1,61 @@
+{
+  stdenv,
+  lib,
+  cmake,
+  pkg-config,
+  flex,
+  bison,
+  lksctp-tools,
+  libidn2,
+  libgcrypt,
+  gnutls,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "freediameter";
+  version = "1.5.0-unstable-2025-03-16";
+
+  src = fetchFromGitHub {
+    owner = "freeDiameter";
+    repo = "freeDiameter";
+    rev = "8e525acdfd439995f3e8e26d5a802fc4ad95d24c";
+    hash = "sha256-ai2R8scP++tdPh303RAl0qdIpehzFoyykAuAyl2w3MA=";
+  };
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    flex
+    bison
+    lksctp-tools
+    libidn2
+    libgcrypt
+    gnutls
+  ];
+
+  configurePhase = ''
+    mkdir -p build
+    cd build
+    cmake ..
+  '';
+
+  buildPhase = ''
+    cmake --build .
+  '';
+
+  installPhase = ''
+    cmake --install . --prefix $out
+  '';
+
+  meta = {
+    description = "Implementation of the Diameter Protocol for Authentication, Authorization, and Accounting";
```

I feel like this is good enough for the short description. If we want to add more details, we can do so in a [meta.longDescription](https://github.com/NixOS/nixpkgs/blob/beddf554ee9fbc87dd5ed9f06f42c9d92f40c704/doc/stdenv/meta.chapter.md#longdescription-var-meta-longdescription)

```suggestion
    description = "Implementation of the Diameter Protocol";
```